.github/workflows/docusaurus.yaml

```diff
@@ -8,11 +8,43 @@ on:
       - 'help/**'
       - 'docusaurus/**'
       - 'js-api/**'
+  pull_request:
+    paths:
+      - 'help/**'
+      - 'docusaurus/**'
+      - 'js-api/**'
 
 jobs:
+  process-merge-commit:
```

Comment:
As it is a common part for ALL workflows lets make it a reusable workflow instead https://docs.github.com/en/actions/using-workflows/reusing-workflows to avoid copy-paste